gradle/javaLibsProject.gradle

```diff
@@ -43,8 +43,6 @@ dependencies {
     dist 'org.bouncycastle:bcpkix-jdk15on:1.61'
 
     dist 'info.picocli:picocli:4.0.1'
-    dist 'org.apache.kafka:kafka-clients:2.3.1'
-    dist 'org.apache.kafka:kafka_2.11:2.3.1'
```

Comment:
Note that, if we are removing these JARs here, we should pack it in the final zip of the module as a 3rd party jar. Otherwise, these jars won't be available in ballerina's final runtime.

Reply:
Yes, added them in the module jar. It's better to remove these from here and add it in the module itself, right?

Reply:
Yes.